### PR TITLE
Small style cleanup

### PR DESCRIPTION
**Pull request by Einh06**
This PR is cleanup small detail regarding `UserPermissions` handling.

### CODE REVIEW

File: schulcloud/UserPermissions.swift [hunk 1]
```diff
@@ -16,6 +16,12 @@ struct UserPermissions : OptionSet {
         self.rawValue = rawValue
     }
     
+    init(array: [String]) {
```
Comment (mathebox) [inline schulcloud/UserPermissions.swift:19]: This can be marked as `convenience`

**Review by mathebox — CHANGES_REQUESTED** [review]
Please also remove https://github.com/schul-cloud/schulcloud-mobile-ios/blob/dev/schulcloud/UserPermissions.swift#L268-L270

File: schulcloud/UserPermissions.swift (hunk 1 above)
Comment (Einh06) [reply schulcloud/UserPermissions.swift:19]: Marking an `init()` as `convenience` in a `struct` is not legal Swift unfortunately.

**Comment by Einh06** [comment]
`init() { rawValue: ( 0, 0 ) }` is required to conform `SetAlgrebra`

**Review by mathebox — APPROVED** [review]

**Pull request merged**